package.json

```diff
@@ -8,6 +8,8 @@
     "build:server": "BUILD_SERVER=true NODE_ENV=production yarn build",
     "build:bookmarklet:dev": "NODE_ENV=development webpack --config src/apps/marklet/webpack.config.js --watch",
     "build:bookmarklet:prod": "NODE_ENV=production webpack --config src/apps/marklet/webpack.config.js",
+    "build:safari-ext:dev": "NODE_ENV=production webpack --config src/extension/safari-extension/webpack.config.js --watch",
```

Comment:
Should `NODE_ENV=production` be set for the `safari-ext:dev` script?

Reply:
No, good catch

Reply:
Actually, easiest to leave this on `production` (i.e. it's going to use the graphql production api endpoint)